### ISSUE TITLE
Setting Node.js/npm Path on a Windows 10 PC

### ISSUE THREAD

**Issue by sus007** [pichillilorenzo/JavaScriptEnhancements#38]
> ### Despite setting the suggested working paths for node & npm executables, I get the following popup every time I use the package.
### Screenshots

![image](https://user-images.githubusercontent.com/7630815/36947288-d11887b6-1ff1-11e8-9a3d-1b69fea754b1.png)
 ### My PATHs
![image](https://user-images.githubusercontent.com/7630815/36947343-6991b166-1ff2-11e8-9a9d-80c2a93fa2b2.png)

> ### Since, there's no official alternative of **nvm** for Windows, so not setting the **PATH** variable still triggers the same popup on startup.

### Specifications

- Sublime Text 3 build: Dev Channel, Build 3157
- OS: Windows 10 Pro Insider (Version 1709, Build 17074.1002)
- JavaScript Enhancements version: 0.15.0

**Comment by sus007** [comment]
Yes, both `node.exe` & `npm.cmd` are in the internal system `PATH` variable of Windows and I did insert the value of `where npm` for `npm_custom_path` & `where node` for `node_js_custom_path`. But, since the documentation states to insert the `PATH` of `nvm` into `PATH` variable and there's no `nvm` for Windows, I also tried leaving your extension's `PATH` variable empty. Still, it shows the same error.
